### PR TITLE
feat: Updating the ft/navbar branch

### PR DESCRIPTION
**Pull request by nsengaqueen**
In this PR we added the following :

1. [x] Renaming the branch from ft-navbar to ft/navbar
2. [x] Deleted the unnecessary files in our branch (Hero, Contact, About, Services)
3. [x] Renamed the Header file name to Navbar 
4. [x] Reducing the non-arbitrary values where possible

### CODE REVIEW

File: src/sections/Navbar.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<nav
+   class="flex flex-row justify-between items-center md:px-8 p-4 lg:p-6 xl:p-6 font-outfit"
+>
+   <div class="flex flex-col text-[1rem] md:text-2xl lg:text-2xl">
```
Comment (IGIRANEZAJosue) [inline src/sections/Navbar.html]: You can remove the 1rem arbitrary value; It doesn't change anything since the text is already 16px by default.

**Comment by christellealexia** [comment]
[Navbar - Responsiveness](https://trello.com/c/tVe2snCg/10-navbar-responsiveness)

File: src/sections/Navbar.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  </div>
+  <div
+    class="absolute lg:flex flex-col items-end hidden md:ml-[20%] lg:ml-0 pt-56 lg:pt-14 pl-32 lg:pl-[88%] font-outfit text-2xl text-green-700 lg:text-black"
+    id="menu"
+  >
+    <p>About</p>
+    <p>Services</p>
+    <p>References</p>
+    <p>Contact</p>
+  </div>
```
Comment (IGIRANEZAJosue) [inline src/sections/Navbar.html:19]: The text for the navbar elements should be bigger and bolder and the gap should be bigger too.

File: Images/Group 4.png
Comment (Algor-Fernand) [inline Images/Group 4.png:1]: The naming here isn't consistent with how we named our files, the extension as well.

File: package.json [hunk 3]
```diff
@@ -2,8 +2,10 @@
   "scripts": {
     "dev": "npx tailwindcss -i ./src/input.css -o ./src/output.css --watch",
     "build": "npx tailwindcss -i ./src/input.css -o ./src/output.css"
-   },
+  },
   "devDependencies": {
+    "prettier": "^3.2.5",
```
Comment (Algor-Fernand) [inline package.json:7]: what's this package for? is it really necessary.

File: src/sections/Navbar.html [hunk 4]
```diff
@@ -0,0 +1,27 @@
+<nav
+  class="flex flex-row justify-between items-center md:px-8 p-4 lg:p-6 xl:p-6 font-outfit  lg:mb-12"
```
Comment (Algor-Fernand) [inline src/sections/Navbar.html:2]: I think there's too much space before the last utility class.

File: .prettierr.js
Comment (Algor-Fernand) [inline .prettierr.js:1]: What is this file for?

**Review by Algor-Fernand — CHANGES_REQUESTED** [review]
The deployment looks good but there are a couple of things I have mentioned that should be changed.

**Review by Algor-Fernand — APPROVED** [review]

**Pull request merged**